docs/query-params.md

```diff
@@ -132,6 +132,7 @@ Only available if `context` equals `fx_desktop_v1`, `fx_desktop_v2`, `fx_desktop
 
 #### Options
 * `sync11`
+* `amo`
```

Comment:
This section is specific to [Sync](https://github.com/mozilla/fxa-content-server/pull/3484/files#diff-9751982e0b5edd17102fe8dfac7e8cb7R83), can you add a new `migration` sub-section in the [OAuth](https://github.com/mozilla/fxa-content-server/pull/3484/files#diff-9751982e0b5edd17102fe8dfac7e8cb7R4) section?